src/test/java/models/User.java

```diff
@@ -0,0 +1,151 @@
+package models;
+
+import com.google.gson.annotations.SerializedName;
+
+public class User {
+
+    @SerializedName("id")
+    private int id;
+    @SerializedName("name")
+    private String name;
+    @SerializedName("username")
+    private String userName;
+    @SerializedName("email")
+    private String email;
+    @SerializedName("address")
+    private Address address;
+    @SerializedName("phone")
+    private String phone;
+    @SerializedName("website")
+    private String website;
+    @SerializedName("company")
+    private Company company;
+
+    class Company{
+        @SerializedName("name")
+        private String name;
+        @SerializedName("catchPhrase")
+        private String catchPhrase;
+        @SerializedName("bs")
+        private String bs;
+    }
+    class Address{
+        @SerializedName("street")
+        private String street;
+        @SerializedName("suite")
+        private String suite;
+        @SerializedName("city")
+        private String city;
+        @SerializedName("zipcode")
+        private String zipCode;
+        @SerializedName("geo")
+        private Geo geo;
+
+
+        class Geo{
```

Comment:
pls format this class by Java standards  
also move inner classes into separate files because class looks too big